vendor/github.com/OpenBazaar/multiwallet/config/config.go

```diff
@@ -12,6 +12,12 @@ import (
 	"golang.org/x/net/proxy"
 )
 
+const (
+	EthereumRegistryAddressMainnet = "0x403d907982474cdd51687b09a8968346159378f3"
+	EthereumRegistryAddressRinkeby = "0x403d907982474cdd51687b09a8968346159378f3"
+	EthereumRegistryAddressRopsten = "0x403d907982474cdd51687b09a8968346159378f3"
+)
```

Comment:
is it a good idea to have the same addresses on all three networks?

Reply:
it seems like you had different addresses for each one of these after all

```golang
"EthereumRegistryAddressMainnet" = "0xab8dd0e05b73529b440d9c9df00b5f490c8596ff",
"EthereumRegistryAddressRinkeby" = "0xab8dd0e05b73529b440d9c9df00b5f490c8596ff",
"EthereumRegistryAddressRopsten" = "0x029d6a0cd4ce98315690f4ea52945545d9c0f460"
```

Reply:
@gubatron - This is a placeholder till the eth wallet testing is complete on Rinkeby. The addresses will be different once this functionality is verified.

Reply:
👍